package/gluon-config-mode-contact-info/Makefile

```diff
@@ -13,10 +13,15 @@ PKG_CONFIG_DEPENDS += $(GLUON_I18N_CONFIG)
 define Package/gluon-config-mode-contact-info
   SECTION:=gluon
   CATEGORY:=Gluon
-  TITLE:=Set a custom string that will be distributed in the mesh.
+  TITLE:=Set a custom string for a contact info that will be distributed in the mesh.
```

Comment:
"for a contact info" sounds weird, "information" is always plural. Also the whole sentence doesn't make a lot of sense.

Maybe "Allows the user to provide contact information to be distributed in the mesh"